crates/pallets/starknet/src/mock.rs

```diff
@@ -70,14 +70,16 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
     let mut t = frame_system::GenesisConfig::default().build_storage::<Test>().unwrap();
 
     let account_class = get_contract_class(ACCOUNT_CONTRACT_PATH);
+    let proxy_class = get_contract_class(include_bytes!("../../../../ressources/proxy_compiled.json"));
+    let arg_account_class = get_contract_class(include_bytes!("../../../../ressources/account_compiled.json"));
     let test_class = get_contract_class(include_bytes!("../../../../ressources/test.json"));
     let l1_handler_class = get_contract_class(include_bytes!("../../../../ressources/l1_handler.json"));
 
     // ACCOUNT CONTRACT
     let contract_address_str = "02356b628D108863BAf8644c945d97bAD70190AF5957031f4852d00D0F690a77";
     let contract_address_bytes = <[u8; 32]>::from_hex(contract_address_str).unwrap();
 
-    let class_hash_str = "025ec026985a3bf9d0cc1fe17326b245dfdc3ff89b8fde106542a3ea56c5a918";
+    let class_hash_str = "025ec026985a3bf9d0cc1fe17326b245dfdc3ff89b8fde106542a3ea56c5a919";
```

Comment:
@EvolveArt this change isn't very descriptive. I'd like to change it before you merge. 

This hash doesn't actually apply to the contract located @ `ACCOUNT_CONTRACT_PATH` so this value essentially just serves as a lookup key, but incrementing it by one is confusing. 

I'll make this more transparant.

Reply:
Right I think we do not need to have such complicated values but rather simple ones like the [blockifier](https://github.com/tdelabro/blockifier/blob/no_std-support/crates/blockifier/src/test_utils.rs#L34) does.

Reply:
Okay attempted to simplify the testing and use actual contract class hashes where applicable.

One thing I don't understand is why blockifier testing isn't using the actual contract_class_hash if they can.

Reply:
Fair question to be honest not sure I know exactly why